README.md

```diff
@@ -246,23 +189,17 @@ We believe that providing a way for hosting providers/managed services to implem
 
 In the long run, we think that demonstrating how we can use existing internet and web standards to make sustainability claims easily discoverable, as well as human and machine readable, will result in a web where it's easier to trust green claims. Not only that, conventions like carbon.txt allows us follow green claims to the supporting evidence used to back them up.
 
-### How can you trust this? What would stop me lying in my carbon.txt file about my site?
-
-This system still requires manual/human verification by Green Web Foundation to prove you are using the hosting provider you say you are. 
-
-You could indeed use carbon.txt to list an already verified green provider in your supply chain and claim it was hosting your site, so your site shows as green through association. However, your site would not show up as green in the tools or dataset until you had been able to submit supporting evidence for manual review that you _really were_ using that provider.
-
-After manual review by our support staff, you would have managed to mark _one_ domain as green.
-
```

Comment:
I've removed this whole question since it's not really relevant at this stage (pilot) and is kinda also answered by the next FAQ anyway.